docs/guide/http_cache.md

```diff
@@ -872,9 +872,41 @@ For custom or eZ Platform controllers still using `X-Location-Id`, a dedicated r
 !!! note
 
     This is currently marked as deprecated. For rendering content it is advised to refactor to use Content View.
-    For other needs there is an internal tag handler in the `ezplatform-http-cache` bundle that can be used,
-    however be aware it will probably change once we move to FOSHttpCache 2.x,
-    so in this case you can stay with `X-Location-Id` for the time being.
+    For other needs there is an FOS tag handler for Twig and PHP that can be used.
+
+##### For custom needs using FOSHttpCache (tagging Relations and more)
```

Comment:
For custom needs **with** FOSHttpCache (tagging Relations and more)

to be consistent with previous title: For responses with X-Location-Id